pkg/test/kube/port_forwarder.go

```diff
@@ -35,7 +35,7 @@ var (
 	// we'l need an index per type.
 	addressMatchIndex = 1
 	forwardRegexIPv4  = regexp.MustCompile(`Forwarding from (127.0.0.1:[0-9]+) -> ([0-9]+)`)
-	forwardRegexIPv6  = regexp.MustCompile(`Forwarding from (\[::1\]:[0-9]+) -> ([0-9]+)`)
+	forwardRegexIPv6  = regexp.MustCompile(`Forwarding from (\[::1]:[0-9]+) -> ([0-9]+)`)
```

Comment:
Ugh. How did this work?

Reply:
It was a redundant escape, so it was semantically equivalent. My change such removes the redundancy.